Documentation/Design/UserStory_WorldInteraction.htm

```diff
@@ -0,0 +1,86 @@
+<html>
+<head></head>
```

Comment:
SpecFlow is for something different.  The goal of a User Story as defined in the design readme, is not the same as the goal of laying out and verifying exact specifications.  The goal here is to help visualize the type of gameplay / features the engine will set us up for out of the box.  Many specs (and regular "Stories" as defied in agile) could certainly be derived from analysis of the User Story, for sure.  (For example, "The user sees that words in an object's description which can be used as targeting keywords are always highlighted in a specific color" could be one.)